project/src/main/java/com/google/sps/data/User.java

```diff
@@ -12,6 +12,7 @@
   private Set<Event> eventsHosting;
   private Set<Event> eventsParticipating;
   private Set<Event> eventsVolunteering;
+  private String imageUrl;
```

Comment:
static? unless editable later

Reply:
I don't think the imageUrl would be static because it's associated with each instance of User